FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/DicomToDataLakeOrchestratorJob.cs

```diff
@@ -54,10 +50,8 @@ public class DicomToDataLakeOrchestratorJob : IJob
             _jobInfo = EnsureArg.IsNotNull(jobInfo, nameof(jobInfo));
             _inputData = EnsureArg.IsNotNull(inputData, nameof(inputData));
             _result = EnsureArg.IsNotNull(result, nameof(result));
-            _dataClient = EnsureArg.IsNotNull(dataClient, nameof(dataClient));
             _dataWriter = EnsureArg.IsNotNull(dataWriter, nameof(dataWriter));
             _queueClient = EnsureArg.IsNotNull(queueClient, nameof(queueClient));
-            _metadataStore = EnsureArg.IsNotNull(metadataStore, nameof(metadataStore));
```

Comment:
Nit: Can you help to update the logs in orchestration job and processing job, like from `FhirToDataLake` to `DicomToDataLake`? So we can avoid modifying same file.

Reply:
Updated.